.vscode/settings.json

```diff
@@ -4,15 +4,15 @@
 
   "cSpell.enabledLanguageIds": ["markdown", "plaintext", "text", "yml"],
 
-  // Use prettier to format TypeScript, JavaScript and JSON files
+  // Use Oxc to format TypeScript, JavaScript and JSON files
   "[typescript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "oxc.oxc-vscode"
   },
   "[javascript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "oxc.oxc-vscode"
   },
   "[json]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "oxc.oxc-vscode"
```

Comment:
This configuration for JSON file formatting is not taking effect. Is it not supported?
@Dunqing

Reply:
Yes